### ISSUE TITLE
Dishorde 2.0.0 echoing weird stuff in its console

### ISSUE THREAD

**Issue by trevorjd** [LakeYS/Dishorde#126]
This "problem" pre-dates 2.0.0. As reported in your Discord on Jan 7th, we're getting odd messages being reported in the Dishorde console. You mentioned it might be mods. We're running Alloc's three, Botman, MeanCloud__EnemyHealth and Zeppz_TopRightChat. 

Might not be super-easy to troubleshoot as I think it takes a while to get into this state. Been running 40 mins now and it's not happening but we only have six players on at the moment. Not a high priority as it doesn't appear to be spilling into game or Discord.

Example: https://i.imgur.com/QWDNYIy.png

**Comment by LakeYS** [comment]
The lines featured in the screenshots are a result of the fix for #119. These are intentional warning messages, so I might suggest moving discussion back to that issue. There will be a future update to suppress some of these warnings.

You mentioned these predate 2.0.0--including the release candidates? This warning was introduced in the release candidates, so if it happened prior, there must have been a different cause.

**Comment by trevorjd** [comment]
Yes, probably came in during the RCs. I'll close this one, thanks.
